app/models/ems_cluster.rb

```diff
@@ -190,7 +190,7 @@ def remove_all_resource_pools
 
   # All RPs under this Cluster and all child RPs
   def all_resource_pools
-    descendants(:of_type => 'ResourcePool')[1..-1].sort_by { |r| r.name.downcase }
+    descendants(:of_type => 'ResourcePool').select { |r| !r.is_default }.sort_by { |r| r.name.downcase }
```

Comment:
@jrafanie that's correct this method is skipping the default resource pool
The next method is [all_resource_pools_with_default](https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_cluster.rb#L196) which includes the default rp (maybe we should rename this one `most_resource_pools` :smile:)

@Fryguy did mention this was caused by the same bug that you linked to in ancestry